src/internal.c

```diff
@@ -25641,6 +25635,30 @@ int wolfSSL_AsyncPush(WOLFSSL* ssl, WC_ASYNC_DEV* asyncDev)
 #endif /* WOLFSSL_ASYNC_CRYPT */
 
 
+/* return the max record size */
+int wolfSSL_GetMaxRecordSize(WOLFSSL* ssl, int maxFragment)
```

Comment:
The `ssl` arg does not have a NULL check. Since this is an internal function and all callers already do a NULL check its okay for now, but something to keep in mind. Note: I like that you avoided the min() use here.